modules/ROOT/pages/quickstart-functions.adoc

```diff
@@ -1,25 +1,57 @@
 = Boson Functions quick start guide
 
-This quick start guide demonstrates how you can create, deploy, and modify a Boson Functions project.
+This quick start guide demonstrates how you can create, deploy, and modify Boson Functions.
 
-.Prerequisites
+== Prerequisites
 
 Before you can complete the steps in the following procedures, you must ensure that:
 
 * You have access to an OpenShift Container Platform cluster, which has the https://docs.openshift.com/container-platform/4.5/serverless/installing_serverless/installing-openshift-serverless.html[OpenShift Serverless Operator], https://docs.openshift.com/container-platform/4.5/serverless/installing_serverless/installing-knative-serving.html#installing-knative-serving[Knative Serving] and https://docs.openshift.com/container-platform/4.5/serverless/installing_serverless/installing-knative-eventing.html#installing-knative-eventing[Knative Eventing] installed.
 * You have installed the https://docs.openshift.com/container-platform/4.5/cli_reference/openshift_cli/getting-started-cli.html#cli-getting-started[OpenShift `oc` CLI].
-* You have installed the https://docs.openshift.com/container-platform/4.5/serverless/installing_serverless/installing-kn.html#installing-kn[Knative `kn` CLI].
 * You must have installed https://docs.docker.com/install/[Docker], and have access to a DockerHub account.
+* You have installed the https://docs.openshift.com/container-platform/4.5/serverless/installing_serverless/installing-kn.html#installing-kn[Knative `kn` CLI]. Installing `kn` provides the binary that enables use of the `kn faas` commands. There is no additional setup required to use `kn faas` commands after `kn` is successfully installed.
```

Comment:
@abrennan89 this is linking to the 4.5 CLI, but I believe `kn faas` is only available in 4.6.

Reply:
4.6 docs are live now so I'll update this